scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -250,29 +254,54 @@ task PostprocessGermlineCNVCalls {
     Int machine_mem_mb = select_first([mem_gb, 7]) * 1000
     Int command_mem_mb = machine_mem_mb - 1000
 
-    String sample_directory = "SAMPLE_${sample_index}"  #this is a hardcoded convention in gcnvkernel
-    String vcf_filename = "${entity_id}.vcf.gz"
+    String genotyped_intervals_vcf_filename = "genotyped-intervals-${entity_id}.vcf.gz"
+    String genotyped_segments_vcf_filename = "genotyped-segments-${entity_id}.vcf.gz"
```

Comment:
Do we not want to provide an option in WDL to not create segments VCF?

Reply:
I think most users would actually care about the segments VCF file anyway (only more hardcore analysts might care about the intervals VCF).